query-algebrizer/src/cc.rs

```diff
@@ -67,16 +67,30 @@ pub enum DatomsColumn {
     ValueTypeTag,
 }
 
+impl DatomsColumn {
+    pub fn as_str(&self) -> &'static str {
+        use DatomsColumn::*;
+        match *self {
+            Entity => "e",
```

Comment:
I think we have two concepts that are a little smushed together:

- Our storage representation.
- Our use of a particular SQL library.

In a sense half of Mentat is the former… but I'd be interested in ideas about encapsulating some of that.